yarn-project/world-state/src/world-state-db/merkle_trees.ts

```diff
@@ -381,8 +382,8 @@ export class MerkleTrees implements MerkleTreeDb {
    * Handles a single L2 block (i.e. Inserts the new commitments into the merkle tree).
    * @param block - The L2 block to handle.
```

Comment:
Missing return desc here ^.

I hope I am not too much of a docs Karen but I think it makes sense to have the description when return is not void.